src/x11/xf86Wacom.c

```diff
@@ -435,18 +437,22 @@ void wcmEmitProximity(WacomDevicePtr priv, bool is_proximity_in,
 void wcmEmitMotion(WacomDevicePtr priv, bool is_absolute, const WacomAxisData *axes)
 {
 	InputInfoPtr pInfo = priv->frontend;
-	int valuators[7];
+	int valuators[9];
```

Comment:
in hindsight, we should really have a `#define` for that number...

Reply:
In the new code, it is only used once, so I think it should be fine.